src/domain/journey/space/SpaceCommunityPage/SpaceCommunityPage.tsx

```diff
@@ -89,6 +100,20 @@ const SpaceCommunityPage = () => {
 
   const [isActivitiesDialogOpen, setIsActivitiesDialogOpen] = useState(false);
 
+  const { data: virtualContributorsData, loading } = useVirtualContributorsMySpacesQuery();
```

Comment:
I don't think you should gather the VCs from 'me->mySpaces'. It's logical to list the VC in the community similar to how we're gathering the other Space community members. 

It should be the contributors to that Space, not specifically part of 'your' spaces.